examples/cms-tina/components/avatar.js

```diff
@@ -0,0 +1,8 @@
+export default function Avatar({ name, picture }) {
+  return (
+    <div className="flex items-center">
+      <img src={picture} className="w-12 h-12 rounded-full mr-4" alt={name} />
```

Comment:
Could we use next/image? 🙏

Reply:
Oh, yup I used your code.... so right back at ya! https://github.com/vercel/next.js/blob/canary/examples/blog-starter/components/avatar.js

Reply:
Yeah we need to update that. Bonus points if you do that here 🙏 🙏🙏

Reply:
You got it! I won't even charge!